### PR TITLE
add release notes to reference docs

### PR DESCRIPTION
**Pull request by glicht**
## Status
Ready

## Related Issues
fixes: link to the issue

## Description
Adding release notes to reference docs.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

**Comment by glicht** [comment]
@yaakovi I think it is enough. We can add more later on if there is need.

**Comment by glicht** [comment]
Preview is available at: https://deploy-preview-107--demisto-content-docs.netlify.app/docs/reference/index#release-notes

**Comment by yaakovi** [comment]
loved the fact that we have a navigator table here for the sections (unlike in Github)

looking at 20.3.3, previous and next are reversed. need to reorder the pages
image

File: content-repo/gendocs.py [hunk 1]
```diff
@@ -263,9 +325,12 @@ def main():
         f.write(index_doc_infos(playbooks_doc_infos, PLAYBOOKS_PREFIX))
         f.write("\n\n## Scripts\n\n")
         f.write(index_doc_infos(script_doc_infos, SCRIPTS_PREFIX))
+        f.write("\n\n## Release Notes\n\n")
```
Comment (Itay4) [inline content-repo/gendocs.py:328]:
```suggestion
        f.write("\n\n## Content Release Notes\n\n")
```

File: content-repo/gendocs.py [hunk 2]
```diff
@@ -263,9 +325,12 @@ def main():
...
+        f.write(index_doc_infos(release_doc_infos, RELEASES_PREFIX, headers=('Name', 'Date')))
```
Comment (Itay4) [inline content-repo/gendocs.py:329]: i dont really like the table to be honest, as there are only two columns with not much content in them at all.. i think a list would be better

**Comment by Itay4** [comment]
the previous of the last release is the latest reference doc before it

can we remove the previous button for the latest release ?

image

**Comment by Itay4** [comment]
i would update the title at the top `Find reference documentation for Integrations, Automations, Playbooks and more.` and add release there

File: content-repo/gendocs.py (hunk 2 above)
Comment (glicht) [reply content-repo/gendocs.py:329]: We can experiment with a different format on another PR. I don't want to over do it in this PR.

**Comment by glicht** [comment]
@yaakovi @Itay4 the buttons at the bottom of prev/next is not something we can easily change. For now we will stay with it as is.

**Pull request merged**